docs/products/kafka/howto/create-topics-automatically.md

```diff
@@ -2,74 +2,86 @@
 title: Create Apache Kafka® topics automatically
```

Comment:
Aiven for Apache Kafka®?

Reply:
For now, I will retain this as is. For topics, we use Apache Kafka topics or for short Kafka topics. Most of this content will be affected later when Governance is enabled and will need updates.